docs/interpreter/livy.md

```diff
@@ -42,7 +42,7 @@ Additional requirements for the Livy interpreter are:
 We added some common configurations for spark, and you can set any configuration you want.
 This link contains all spark configurations: http://spark.apache.org/docs/latest/configuration.html#available-properties.
```

Comment:
If you don't mind, can we change this part to like below? Then ppl don't need to copy & paste manually. 
```
You can find all Spark configurations in [here](http://spark.apache.org/docs/latest/configuration.html#available-properties).
```